tests/unit/runloop-test.js

```diff
@@ -37,4 +37,36 @@ test('"run.next" works with arguments', function(assert) {
     obj.nextMe('wat');
     done();
   }, 20);
+});
+
+test('"run.debounce" works with es6 class', function(assert) {
+  let done = assert.async();
+  assert.expect(1);
+
+  class Foo {
+    @debounce(50)
+    nextMe() {
+      assert.ok(true);
+      done();
+    }
+  }
+
+  let obj = new Foo();
+  obj.nextMe();
```

Comment:
It doesn't look like we're testing the debounce functionality here, can we call these functions at least twice to make sure they're working properly?